plugins/ipam/dhcp/lease.go

```diff
@@ -344,6 +360,45 @@ func (l *DHCPLease) maintain() {
 	}
 }
 
+func (l *DHCPLease) checkLinkExistsWithBackoff(ctx context.Context) (bool, error) {
+	checkFunc := func() (bool, error) {
+		_, err := netlink.LinkByName(l.link.Attrs().Name)
+		if err != nil {
+			var linkNotFoundErr *netlink.LinkNotFoundError = &netlink.LinkNotFoundError{}
+			if errors.As(err, linkNotFoundErr) {
+				return false, nil
+			}
+			return false, err
+		}
+		return true, nil
+	}
+
+	ctx, cancel := context.WithTimeout(ctx, linkCheckTotalTimeout)
+	defer cancel()
+
+	exists, err := backoffRetryLinkExists(ctx, linkCheckRetryMax, checkFunc)
```

Comment:
`linkCheckRetryMax` is constant variable, defined as global scope in this file. So we may remove it from function argument. Otherwise, remove constant variable, `linkCheckRetryMax` and define it in function variable.

Reply:
thanks, simplified so that it isn't passed through the method param and just used directly there, good call